src/Opdex.Platform.Application/EntryHandlers/Tokens/CreateTokenCommandHandler.cs

```diff
@@ -37,20 +37,25 @@ public async Task<ulong> Handle(CreateTokenCommand request, CancellationToken ca
         }
 
         var summary = await _mediator.Send(new CallCirrusGetStandardTokenContractSummaryQuery(request.Token,
-            request.BlockHeight,
-            includeBaseProperties: true,
-            includeTotalSupply: true));
+                                                                                              request.BlockHeight,
+                                                                                              includeBaseProperties: true,
+                                                                                              includeTotalSupply: true));
 
         token = new Token(request.Token,
-            summary.Name,
-            summary.Symbol,
-            (int)summary.Decimals.GetValueOrDefault(),
-            summary.Sats.GetValueOrDefault(),
-            summary.TotalSupply.GetValueOrDefault(),
-            request.BlockHeight);
+                          summary.Name,
+                          summary.Symbol,
+                          (int)summary.Decimals.GetValueOrDefault(),
+                          summary.Sats.GetValueOrDefault(),
+                          summary.TotalSupply.GetValueOrDefault(),
+                          request.BlockHeight);
 
-        tokenId = await _mediator.Send(new MakeTokenCommand(token, request.BlockHeight));
-        if (tokenId == 0) _logger.LogError("Something went wrong indexing the token");
+        tokenId = await _mediator.Send(new MakeTokenCommand(token, request.BlockHeight), CancellationToken.None);
+
+        if (tokenId == 0)
+        {
+            _logger.LogError("Something went wrong indexing the token");
```

Comment:
I know that we weren't already, but we should log the token address as a custom property

Reply:
Added this, there are many other logs that could use some updates similar to this as well. I didn't do them but we can come back.